payments/card/other-features.md

```diff
@@ -74,6 +74,16 @@ when="at the 3-D Secure verification for credit card payments" full_reference=tr
 
 {% include prices.md %}
 
+## 3D Secure 2.0
+
+When dealing with card payments, 3D Secure authentication of the
+cardholder is an essential topic. 3D Secure 2.0 is an improved version of the
+old protocol, now allowing frictionless payments where transactions can be
+completed without input from the cardholder. Therefore, there are certain fields
+that should be included when implementing 3D Secure 2.0. These are listed below: 
+*   `payment.riskIndicator`
+*   `payment.cardholder`
```

Comment:
Perhaps we should enlist all fields within `riskIndicator` and `cardHolder` by just providing a full JSON code example?